tensorboard/plugins/scalar/tf_scalar_dashboard/tf-scalar-dashboard.html

```diff
@@ -286,9 +288,67 @@
         });
       },
 
-      _makeCategories(runToTag, selectedRuns, tagFilter) {
+      _makeCategories(runToTagInfo, selectedRuns, tagFilter) {
+        const runToTag = _.mapValues(runToTagInfo, x => Object.keys(x));
         return categorizeTags(runToTag, selectedRuns, tagFilter);
       },
+      _tagMetadata(runToTagInfo, runs, tag) {
+        /*
+         * Each run--tag combination may have its own display name and
+         * description, but we display just one chart per tag: this is a
```

Comment:
The PR curve plugin will face this issue too. I might move this logic later to somewhere else.